Assets/css/style.css

```diff
@@ -0,0 +1,33 @@
+/* Navigation Bar Styles */
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  width: 200px;
+  background-color: #f1f1f1;
+}
+
+li a {
+  display: block;
+  color: #000;
+  padding: 8px 16px;
+  text-decoration: none;
+}
+
+/* Change the link color on hover */
+li a:hover {
```

Comment:
Please replace "li a:hover" with "nav ul li a:hover" just so it doesnt effect all LI's on the page